usagov/vote-gov-drupal
# Vote-925 -> language switcher test update

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-925](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-924&assignee=611272e0eb7aef00694b8a97)

## Description (optional)
This PR will update `translations-url` test and add it to run through template pages along with checking if drop-down is visual once clicked

## Deployment and testing (required, if applicable)
### Testing steps
1. Pull down branch 
2. start local 
3. cd into `testing` folder 
4. run `npm run cy:open`
5. select `translations-urls.cy.js` test and verify that it is passing
